src/transport/AdminPairingTable.h

```diff
@@ -29,8 +29,11 @@ namespace chip {
 namespace Transport {
 
 typedef uint16_t AdminId;
-static constexpr AdminId kUndefinedAdminId = UINT16_MAX;
+typedef uint64_t FabricId;
 
+static constexpr AdminId kUndefinedAdminId   = UINT16_MAX;
+// static constexpr FabricId kUndefinedFabricId = UINT64_MAX;
+static constexpr uint16_t kUndefinedVendorId = UINT16_MAX;
```

Comment:
This is an odd value to use...  the spec has 0xFFFF as a wildcard vendor id and 0 as a "no vendor id specified" (i.e. the vendor is CHIP).  Do we want the wildcard meaning here?

Reply:
change it to 0U and stored in PeerId.h alongside kUndefinedFabricId and kUndefinedNodeId